app/models/article.rb

```diff
@@ -0,0 +1,13 @@
+class Article < ApplicationRecord
+  include Visible
+
+  belongs_to :owner, class_name: 'User'
+  has_many :comments, dependent: :destroy
```

Comment:
@sansuaza what the **dependent** means?

Reply:
means that if the articles it belongs to is deleted, this command will also be deleted